fastboot-tests/index-test.js

```diff
@@ -0,0 +1,49 @@
+var expect = require('chai').expect;
+
+function serializeMediaQueries($) {
+  let data = {};
+
+  $('#media-queries dt').toArray().forEach(function(dt) {
+    data[$(dt).text().trim()] = $(dt).next().text().trim();
+  });
+
+  return data;
+}
+
+describe('index', function() {
+
+  it('renders', function() {
+    return this.visit('/')
+      .then(function(res) {
+        var $ = res.jQuery;
+        // var response = res.response;
+
+        expect($('body').length).to.equal(1);
+        expect($('h1').text().trim()).to.equal('ember-fastboot-addon-tests');
+        expect($('#dimensions').length).to.equal(1);
+        expect($('#media-queries').length).to.equal(1);
+      });
+  });
+
+  it('can stub media features', function() {
+    return this.visit('/')
+      .then(function(res) {
+        var $ = res.jQuery;
+        // var response = res.response;
+
+        expect($('#width').text().trim()).to.equal('900'); // see fixtures/fastboot/app/instance-initializers/fastboot/stub-media.js
+        expect(serializeMediaQueries($)).to.deep.equal({
+          isSmallAndUp: "true",
+          isMediumAndUp: "true",
+          isLargeAndUp: "false",
+          isExtraLargeAndUp: "false",
+
+          isExtraSmallAndDown: "false",
+          isSmallAndDown: "false",
+          isMediumAndDown: "true",
+          isLargeAndDown: "true"
+        });
+      });
+  });
+
+});
```

Comment:
Github is showing the "no newline" warning at the end of this file.

<img width="427" alt="screen shot 2017-02-25 at 2 30 26 pm" src="https://cloud.githubusercontent.com/assets/15169/23335284/078bc426-fb67-11e6-897e-b3c5479b9fd1.png">